elephant/conversion.py

```diff
@@ -1267,6 +1267,26 @@ def discretise_spiketimes(spiketrains, sampling_rate):
     -------
     neo.SpikeTrain or list of neo.SpikeTrain
         The discretised spiketrain(s)
+
+    Examples
+    --------
+    >>> import neo
+    >>> import numpy as np
+    >>> import quantities as pq
+    >>> from elephant import conversion
+    >>>
+    >>> times = (np.arange(10) + np.random.uniform(size=10)) * pq.ms
```

Comment:
Without a fixed seed this will fail when being run as a doctest since the result will change every time.
What's our policy for these examples? Should they work as doctests? 
I guess that would help to keep the examples up to date, when running this I noticed that a few other examples in this file currently fail as well.

Reply:
fixed, see https://github.com/NeuralEnsemble/elephant/pull/468/commits/232dcb0558b171a083e88a88dd0639b68d5a30ed